src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+
+function getEmployees(list) {
+  const employees = [...list.children];
+
+  return employees;
+}
+
+function sortList(list) {
+  const convertSalary = (el) => (
+    +el.dataset.salary.replace(/[$,]/g, '')
+  );
+
+  const salary = getEmployees(listOfEmployees).sort((a, b) => (
+    convertSalary(b) - convertSalary(a)
+  ));
+
+  list.innerHTML = `
+    ${salary.map(employee => `
+      <li
+        data-position="${employee.dataset.position}"
+        data-salary="${employee.dataset.salary}"
+        data-age="${employee.dataset.age}"
+      >
+        ${employee.innerText}
+      </li>
+    `).join('')}
+  `;
+}
+
+getEmployees(listOfEmployees);
+sortList(listOfEmployees);
```

Comment:
All you actually need is one call
```suggestion
sortList(listOfEmployees);
```

Reply:
4.Call both functions.